src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/ContinuousVariableCard.js

```diff
@@ -496,19 +496,27 @@ export default compose(
       props.variable.isSurvivalCustom === nextProps.variable.isSurvivalCustom
     ),
     ({
-      defaultData: { bins },
+      binsAreCustom,
+      defaultData: { bins: defaultBins },
       dispatch,
       fieldName,
       id,
+      variable: { bins },
```

Comment:
```suggestion
```

Reply:
This line is necessary for line 518 to work.